packages/react-core/src/filters/Filter.js

```diff
@@ -57,15 +57,23 @@ function passesFilter(columns, filters, feature) {
 }
 
 export function buildFeatureFilter({ filters = {}, type = 'boolean' }) {
-  if (!Object.keys(filters).length) {
+  const columns = Object.keys(filters);
+
+  if (!columns.length) {
     return () => (type === 'number' ? 1 : true);
   }
 
   return (feature) => {
-    const columns = Object.keys(filters);
     const f = feature.properties || feature;
     const featurePassesFilter = passesFilter(columns, filters, f);
 
     return type === 'number' ? Number(featurePassesFilter) : featurePassesFilter;
   };
 }
+
+// Apply certain filters to a collection of features
+export function applyFilters(features, filters) {
```

Comment:
I think that it's not necessary, you should check on "parent" component if need to apply filters or not. Here you're calling to applyFilters but it's not applying anything necessarily

Reply:
> I think this can be used in advanced use cases.

I've just read it, I think that it could be added in the future if needed. But for now I'll try to move this logic to the parent component instead.

Reply:
Why in the future but not now? I think it's a wrapper to simplify applying C4R filters. I think that the only thing that we can change in the future is to remove the underscore sign and export it to every user. In the meanwhile, even with the underscore, I think it's something that can be useful in PS and we can avoid the `.filter` and the `buildFeatureFilter`, you just pass a feature collection and C4R filters and the magic is done. If, in the future, we want to change how buildFeatureFilter is made or called, I think it's better if we have this wrapper because applyFilters interface won't change.

Reply:
I'm not talking about `applyFilters` function, I'm talking about checking if the features must be filtered or not. My suggestion was to move the condition to the component that is calling to `applyFilters`, anyways.... I think that you're right, you should check it always in this function.

LGTM